apps/cnp/sbox/base/descheduler2.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: descheduler2
+  namespace: cnp
+spec:
+  timeout: 20m
+  releaseName: descheduler
+  chart:
+    spec:
+      chart: descheduler
+      version: 0.18.0
+      sourceRef:
+        kind: HelmRepository
+        name: descheduler
+        namespace: cnp
+  interval: 1m
+  values:
+    kind: CronJob
+    image:
+      repository: registry.k8s.io/descheduler/descheduler
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 500m
+        memory: 256Mi
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+      privileged: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      runAsUser: 1000
```

Comment:
I cant see this stuff in helm chart values? Looks like default code from the job spec. Can remove? There's probably a bunch more stuff that was in the YAML output of the resource but not configurable via values this way